### PR TITLE
Add gulp and package updates to release

### PR DESCRIPTION
**Pull request by digeff**
Add command needed to build this package for VS

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,7 +77,8 @@
     "postinstall": "node ./node_modules/vscode/bin/install",
     "patch": "npm version patch -m '%s'",
     "minor": "npm version minor -m '%s'",
-    "major": "npm version major -m '%s'"
+    "major": "npm version major -m '%s'",
+    "package": "gulp package"
```
[the remark this answers is not in the document]
Comment (digeff) [reply package.json]: Normally we should. These changes are only for build/compile time, and they shouldn't affect anything that we release. I don't want to change package-lock.json in this commit and potentially change one of the version of the dependencies that we used in VS 15.7 originally

File: gulpfile.js [hunk 2]
```diff
@@ -141,7 +143,12 @@ gulp.task('vsce-publish', function () {
     return vsce.publish();
 });
 gulp.task('vsce-package', function () {
-    return vsce.createVSIX();
+    const cliOptions = minimist(process.argv.slice(2));
```
[the remark this answers is not in the document]
Comment (digeff) [reply gulpfile.js]: I cherry-picked this change from the master branch, so I'd rather leave them as-is. Feel free to send a PR to master with any improvements you want to make.

File: gulpfile.js [hunk 3]
```diff
@@ -14,6 +14,8 @@ const nls = require('vscode-nls-dev');
 const vsce = require('vsce');
 const es = require('event-stream');
 const runSequence = require('run-sequence');
+const del = require('del');
```
[the remark this answers is not in the document]
Comment (digeff) [reply gulpfile.js]: This was part of a commit I cherry-picked from master. I don't think it's currently used but I'd rather not change anything else.

File: gulpfile.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (roblourens) [reply gulpfile.js]: 2 is taking the args after `gulp vsce-package ...`

**Pull request merged**